keps/sig-architecture/2019-03-19-go-modules.md

```diff
@@ -197,8 +206,9 @@ Compatibility implications:
 * breaking go changes in each release impact consumers that have not pinned to particular tags/shas (as they do today)
 * conflicting version requirements (direct or transitive) can result in impossible-to-build or impossible-to-update dependencies (as they do today)
 
-Allowed versioning changes:
+Allowed future versioning changes:
 * modules published this way can transition to semantic import versioning in the future
+* modules published this way can transition to v1.x.y semver tagging in the future (with go API compatibility guarantees)
```

Comment:
If we're locked into 1 always being the major digit, we will never be able to remove or modify things. That seems like an impossible restriction for our code.

Reply:
I should rephrase this. The v0.x.y tags wouldn't prevent us from moving to v1.x.y in the future. Other considerations (like the point you raise) probably would. I just wanted to point out this specific proposal wasn't limiting future options.

Reply:
OK, that's probably worth clarifying.

Reply:
clarified